app/views/recipes/_recipe.html.erb

```diff
@@ -1,33 +1,54 @@
-<div id="<%= dom_id recipe %>">
-  
-  <p>
-    <strong>Name:</strong>
-    <%= recipe.name %>
-  </p>
+<div class="d-flex justify-content-center flex-column">
+    <h1 class="text-center"><%= @recipe.name%></h1>
+    <div id="recipe-details" class="flex">
+      <div>
+        <div>
+          <h3>
+            Preparation time: <%=@recipe.preparation_time %>
+          </h3>
+          <h3>
+            Cooking time: <%=@recipe.cooking_time %>%
+          </h3>
+          </h3>
+        </div>
+        <div>
+          <!-- ruby on rails checkbox activate or deactivate public mode -->
+          <label for="public">Public</label><%= check_box_tag "public", "1", false, data: { toggle: "toggle" } %>
```

Comment:
note: make sure that this check_box_tag is changing the database public field. Maybe we need to put it in a form submission